src/accountsdb/download.zig

```diff
@@ -408,8 +423,44 @@ fn checkCode(code: curl.libcurl.CURLcode) !void {
     return error.Unexpected;
 }
 
-pub fn setNoBody(self: curl.Easy, no_body: bool) !void {
-    try checkCode(curl.libcurl.curl_easy_setopt(self.handle, curl.libcurl.CURLOPT_NOBODY, @as(c_long, @intFromBool(no_body))));
+fn setNoBody(self: curl.Easy, no_body: bool) !void {
+    try checkCode(curl.libcurl.curl_easy_setopt(
+        self.handle,
+        curl.libcurl.CURLOPT_NOBODY,
+        @as(c_long, @intFromBool(no_body)),
+    ));
+}
+
+fn setProgressFunction(
```

Comment:
For now this makes sense here. What do you think about upstreaming these functions? Then later we could remove them from here.

Reply:
Yeah don't mind. The quality of the library is questionable, but it serves us well so far. I don't really see why we need to have a wrapper library around this TBH, we could write our own wrapper better and simpler.